### ISSUE TITLE
2.10.1: Dictionary View doesn’t show

### ISSUE THREAD

**Issue by Moonbase59** [phibr0/obsidian-dictionary#24]
When trying to open the dictionary side pane, I get:

```text
app.js:1 Failed to open view TypeError: _a.getElementById is not a function
    at append_styles (eval at <anonymous> (app.js:1), <anonymous>:529:75)
    at add_css$1 (eval at <anonymous> (app.js:1), <anonymous>:2284:2)
    at init (eval at <anonymous> (app.js:1), <anonymous>:1003:22)
    at new DictionaryView$1 (eval at <anonymous> (app.js:1), <anonymous>:2923:3)
    at DictionaryView.eval (eval at <anonymous> (app.js:1), <anonymous>:3052:26)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:31:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app.js:1), <anonymous>:27:12)
    at DictionaryView.onOpen (eval at <anonymous> (app.js:1), <anonymous>:3051:16)
(anonymous) @ app.js:1
```

![Workspace 1_160](https://user-images.githubusercontent.com/3706922/126680921-2a96979e-ba51-4c9e-bdb7-09ef1b109c0d.png)

**Comment by phibr0** [comment]
I am not entirely sure why that happened but it seems to be fixed now

**Comment by phibr0** [comment]
No it's not fixed.. I don't understand what is happening

**Comment by Moonbase59** [comment]
Can you reproduce? Anything I could do or test?

**Comment by phibr0** [comment]
Yes, but not consistently

**Comment by Moonbase59** [comment]
Too bad. I can’t even make it appear on two different machines. Tried restarting Obsidian, closing/reopening the side pane, deleting the cache, etc.

Btw, it shows version 2.10.1 and wants to upgrade to 2.10.2 which fails.

What’s better for testing?

- revert back to 2.10.0, or
- stick at 2.10.1 and look through your commits?

**Comment by Moonbase59** [comment]
Did a quick check, comparing 2.10.0 → 2.10.2 ([Compare](https://github.com/phibr0/obsidian-dictionary/compare/2.10.0...2.10.2)) but nothing obvious.

Except maybe [b251756](https://github.com/phibr0/obsidian-dictionary/commit/b251756ee8993b91574b91763a93786b3574b5b3)—sure you committed everything? "Fix View not loading" is just a version number change in `manifest.json`?

**Comment by phibr0** [comment]
I fixed the manifest so it doesn't want to update anymore. Not really sure what caused it but I will look through my commits again.

> Except maybe b251756—sure you committed everything? "Fix View not loading" is just a version number change in manifest.json?

I guess it didn't happen once and I just thought "oh nice, guess I fixed it" - I didn't even realize I didn't change anything

**Comment by phibr0** [comment]
I tested every single commit I made and every build had this error, I even tried using the Commit where I first introduced the Caching Feature using `git checkout 83b42344d01e666333fb8643a9671f2c156bc58d`.

Even this Build has the same Error now. I also deleted the untracked `data.json`, which didnt help either.

**Comment by Moonbase59** [comment]
That’s really weird.

**Comment by phibr0** [comment]
https://github.com/sveltejs/svelte/issues/6567

Same Issue that is happening with this Plugin. When downgraded to an older svelte version it works fine.

**Comment by Moonbase59** [comment]
Darn, so a framework issue. Then again, since svelte is kinda compiler and not a module, it might be worth downgrading until that’s fixed, so at least you could release new versions. What do you think?

**Comment by phibr0** [comment]
Yeah already did that, its not a Problem and I actually didn't even mean to upgrade it. I just ran `npm upgrade` to get Obsidians new request API, but the command of course upgrades every package. I need to be more careful with stuff like that.

**Comment by phibr0** [comment]
Seems like its still happening, can't even reproduce my fix from a few hours ago

**Comment by phibr0** [comment]
Seems like this Issue is finally fixed as of Svelte v3.40.3 (https://github.com/sveltejs/svelte/issues/6567)

Closing this now

**Comment by Moonbase59** [comment]
Confirm working fine with version 2.13.2.
